### ISSUE TITLE
Readiness probe fails on Sidekiq 7

### ISSUE THREAD

**Issue by drqCode** [sidekiq/sidekiq#5782]
Ruby version:
ruby 3.1.3p185
Rails version:
7.0.4.2
Sidekiq / Pro / Enterprise version(s):
7.0.3/7.0.6

K8s readiness probe fails due to incompatibility between new Sidekiq 7 and https://rubygems.org/gems/sidekiq-monitor-stats. 
Here's the error when running `REDIS_URL=**REDACTED**  bundle exec sidekiqmon | grep $(hostname)`:
```
Couldn't get status: undefined method `join' for {"image"=>0}:Hash

            process["weights"].sort_by { |queue| queue[0] }.map { |queue| queue.join(": ") }
```

@mperham `sidekiq-monitor-stats` seems to be deprecated. Do you have an advice on how to best approach K8s probes for Sidekiq 7?

**Comment by drqCode** [comment]
I made a mistake. It's not https://rubygems.org/gems/sidekiq-monitor-stats, but only Sidekiq 7 :)

**Comment by mperham** [comment]
Try Sidekiq 7.0.2. This will be fixed in 7.0.4.

**Comment by mperham** [comment]
7.0.4 has been released
